src/styles/blocks/header.scss

```diff
@@ -0,0 +1,21 @@
+/* #region header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  max-width: 100%;
```

Comment:
The 'max-width' property is not necessary since the default value is already 100% for block elements.